fileversion.go

```diff
@@ -218,3 +240,57 @@ func (self *FileVersion) GetFixedFileInfo() (*FixedFileInfo, error) {
 		FileSubtype: VerFileSubtype(ffi.FileSubtype),
 	}, nil
 }
+
+func (self *FileVersion) GetFileInfoTranslations() ([]FileInfoTranslation, error) {
+	var fit *FileInfoTranslation
+	var len uint32
+	err := wrappers.VerQueryValue(
+		&self.data[0],
+		syscall.StringToUTF16Ptr(`\VarFileInfo\Translation`),
+		(**byte)(unsafe.Pointer(&fit)),
+		&len)
+	if err != nil {
+		return nil, NewWindowsError("VerQueryValue", err)
+	}
+
+	result := make([]FileInfoTranslation, 0)
+	if len == 0 {
+		return result, nil
+	}
+
+	ti := fit
+	for i := 0; i < int(len/uint32(unsafe.Sizeof(*ti))); i++ {
+		result = append(result, FileInfoTranslation{Language: ti.Language, CodePage: ti.CodePage})
+		ti = (*FileInfoTranslation)(unsafe.Pointer(uintptr(unsafe.Pointer(ti)) + unsafe.Sizeof(*ti)))
+	}
+	return result, nil
+}
+
+func (self *FileVersion) GetStringFileInfo(translation FileInfoTranslation, stringName string) (string, error) {
+	var offset *uint16
+	var len uint32
+	err := wrappers.VerQueryValue(
+		&self.data[0],
+		syscall.StringToUTF16Ptr(fmt.Sprintf("\\StringFileInfo\\%04x%04x\\%s", translation.Language, translation.CodePage, stringName)),
+		(**byte)(unsafe.Pointer(&offset)),
+		&len)
+	if err != nil {
+		return "", NewWindowsError("VerQueryValue", err)
+	}
+	if len == 0 {
+		return "", nil
+	}
+	return LpstrToString(offset), nil
+}
+
+func (self *FileVersion) GetFirstStringFileInfo(stringName string) (string, error) {
+	tr, err := self.GetFileInfoTranslations()
+	if err != nil || len(tr) == 0 {
+		return "", err
+	}
+	return self.GetStringFileInfo(tr[0], stringName)
+}
+
+func (self *FileVersion) GetStandardStringFileInfo(stringName StringFileInfoNames) (string, error) {
```

Comment:
Any reason to have separate methods for string vs. the enumeration as opposed to having everything take the enumeration?  Since Go doesn't prevent you from casting a string to the enumeration type, I'd think that a single method should do the job.